src/mlpack/tests/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ add_executable(mlpack_test
   sparse_coding_test.cpp
   spill_tree_test.cpp
   split_data_test.cpp
+  string_utility_test.cpp
```

Comment:
Do you want to also add `main_tests/preprocess_string_test.cpp` to test that the different options of `preprocess_string_test` seem to work correctly?